MetaMask/metamask-extension
# Activate LavaMoat scuttling security feature

## Explanation

Activating new LavaMoat security feature introduced and explained in https://github.com/lavamoat/lavamoat/pull/360

## Manual Testing Steps

The app should function just the same as before, the only change you should see is that when you try accessing scuttled properties of the `window` object you should see an error denying you from accomplishing that.

You can achieve that by simply testing that out in the console.

### Before

```javascript
window.alert(1); // works
```

### After

```javascript
window.alert(1); // throws an error of how `alert` is a scuttled property
```

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
